src/components/project/project.service.ts

```diff
@@ -792,6 +805,32 @@ export class ProjectService {
       nodevar: 'project',
     });
 
+    const budgets = await this.budgetService.list(
+      {
+        filter: {
+          projectId: input.id,
+        },
+      },
+      session
+    );
+
+    const pendingBudget = budgets.items.find(
+      (b) => b.status === BudgetStatus.Pending
+    );
+    //574 -The pending budget should be set to active i.e Current when the project gets set to active
+    if (
+      (changes.status.includes(ProjectStatus.InDevelopment) ||
+        changes.status.includes(ProjectStatus.Pending)) &&
+      pendingBudget?.status.includes(BudgetStatus.Pending)
+    ) {
```

Comment:
I know this will be changing with the event code. But until then, it's when the project becomes _active_. The comment is correct but the logic is wrong.
```suggestion
    //574 -The pending budget should be set to active i.e Current when the project gets set to active
    if (changes.status.includes(ProjectStatus.Active) && pendingBudget) {
```

Reply:
Thank you @CarsonF for correcting logic.